controller/src/test/java/io/pravega/controller/store/stream/StreamMetadataStoreTest.java

```diff
@@ -427,6 +427,41 @@ public void scaleTest() throws Exception {
     }
 
     @Test
+    public void updateTest() throws Exception {
+        final String scope = "ScopeUpdate";
+        final String stream = "StreamUpdate";
+        final ScalingPolicy policy = ScalingPolicy.fixed(2);
+        final StreamConfiguration configuration = StreamConfiguration.builder().scope(scope).streamName(stream).scalingPolicy(policy).build();
+
+        long start = System.currentTimeMillis();
+        store.createScope(scope).get();
+
+        store.createStream(scope, stream, configuration, start, null, executor).get();
+        store.setState(scope, stream, State.ACTIVE, null, executor).get();
+
+        // region idempotent
+        final StreamConfiguration configuration2 = StreamConfiguration.builder().scope(scope).streamName(stream).scalingPolicy(policy).build();
```

Comment:
Trying to update with same configuration, maybe you can try with a different config to exercise different flows?